dali/kernels/imgproc/roi.h

```diff
@@ -89,7 +89,7 @@ TensorShape<ndims> ShapeFromRoi(const Roi<spatial_dims> &roi, int nchannels) {
  * Convenient overload for batch processing (creating TensorListShape)
  */
 template <int spatial_dims, int ndims = spatial_dims + 1>
-TensorListShape<ndims> ShapeFromRoi(const std::vector<Roi<spatial_dims>> &rois, int nchannels) {
+TensorListShape<ndims> ShapeFromRoi(span<const Roi<spatial_dims>> rois, int nchannels) {
   DALI_ENFORCE(!rois.empty(), "Provided argument doesn't contain any Roi");
```

Comment:
Empty TensorListShape is perfectly valid - why throw?

Reply:
Done